leshan-core/src/main/java/org/eclipse/leshan/core/model/ObjectLoader.java

```diff
@@ -107,6 +105,52 @@ public static ObjectModel loadDdfFile(InputStream input, String streamName) {
         return models;
     }
 
+    /**
+     * Load object definition from DDF files.
+     * 
+     * @param path directory path to the DDF files
+     * @param filenames names of all the DDF files
+     */
+    public static List<ObjectModel> loadDdfFiles(String path, String[] filenames) {
+        List<ObjectModel> models = new ArrayList<>();
+        for (String filename : filenames) {
+            String fullpath = path + filename;
```

Comment:
I changed the code to support trailing `/`.
I renamed `loadDdfFiles` in `loadDdfResources` to be more explicit.
I also add some javadoc and link to `class.getResourceAsStream`